modules/ding_variables/ding_variables.info

```diff
@@ -0,0 +1,6 @@
+name = Ding Variables
+description = Exposed custom variables as a JSON format.
+core = 7.x
+package = Ding!
+version = 7.x-3.0.5
```

Comment:
Normally we don't have version tags inside the custom module files.

Reply:
Also there are no dependencies mentioned here. That does not seem right.